contents/blog/posthog-vs-hotjar.mdx

```diff
@@ -10,626 +10,259 @@ featuredImage: >-
 
-In this guide, we'll cover:
+### 2. We support both web and mobile apps
 
-- [Hotjar and PostHog feature comparison](#core-features)
-- [Session replays](#session-replays) 
-- [Heatmaps](#heatmaps)
-- [Product analytics](#product-analytics)
-- [In-app messages and user research](#in-app-messages-and-user-research)
-- [Apps and integrations](#apps-and-integrations)
-- [Pricing](#how-much-does-hotjar-cost)
+We offer integrations for all the most popular web and and mobile SDKs, including [Next.js](/docs/libraries/next-js), [React](/docs/libraries/react), [React Native](/docs/libraries/react-native), [Android](/docs/libraries/android), [iOS](/docs/libraries/ios), and [Flutter](/docs/libraries/flutter). Unlike Hotjar, we also support [session replay on mobile apps](/session-replay/mobile) through our iOS and Android SDKs – we're working on React Native and Flutter support, too. This means you can track all versions of your app using PostHog.
 
-## How is PostHog different?
+### 3. We're cheaper and have a generous free tier
 
-### 1. PostHog is an all-in one platform
+Every customer gets 5,000 recordings and 1 million events for free each month, no matter what plan they're on. Our generous free tier means more than 90% of companies use PostHog for free, and you'll pay less when you do use up your free credits. 
 
-Hotjar focuses mainly on session replays, heatmaps, and user research. That means you need to adopt additional tools for things like [product analytics](/product-analytics), [feature management](/feature-flags), and [A/B testing](/ab-testing). PostHog integrates all these features into one platform (in addition to [session replays and heatmaps](/session-replay)).
+See the [price comparison section](#price-comparison) for more.
 
-### 2. PostHog is built for engineers
+## Product comparison
 
-PostHog integrates powerful features that aren’t available in Hotjar and which are designed for technical users, such as feature flags with JSON payloads, which enable you to trigger in-app messages, or other notifications. 
+The best way to imagine PostHog is as an [alternative to Hotjar](/blog/best-hotjar-alternatives), [Heap](/blog/best-heap-alternatives), and [LaunchDarkly](/blog/best-launchdarkly-alternatives) rolled into one.
 
-We built PostHog to support technically-savvy product managers and engineers – especially [engineers with a product focus in their role](/blog/what-is-a-product-engineer). Anyone can use PostHog ([just ask our marketing team](/blog/posthog-marketing)), but our primary goal is to give engineers the tools they need to ship impactful features at speed.
+It's common to use Hotjar alongside some combination of Mixpanel, Amplitude, Heap and Google Analytics. Switching to PostHog eliminates all that mess.
```

Comment:
This kind of says the same thing as the last sentence, maybe only one sentence that mentions competitors?

Reply:
fair